src/vs/workbench/api/common/extHostChatAgents2.ts

```diff
@@ -68,31 +69,36 @@ class ChatAgentResponseStream {
 				}
 			}
 
-			const _report = (progress: IChatProgressDto, task?: (progress: vscode.Progress<vscode.ChatResponseWarningPart | vscode.ChatResponseReferencePart>) => Thenable<string | void>) => {
+			const _report = (progress: IChatProgressDto, task?: (progress: vscode.Progress<vscode.ChatResponseWarningPart | vscode.ChatResponseReferencePart>) => Thenable<string | void>, progressReporterHandle?: number) => {
 				// Measure the time to the first progress update with real markdown content
 				if (typeof this._firstProgress === 'undefined' && 'content' in progress) {
 					this._firstProgress = this._stopWatch.elapsed();
 				}
 
-				this._proxy.$handleProgressChunk(this._request.requestId, progress)
-					.then((handle) => {
-						if (handle) {
-							task?.({
-								report: (p) => {
+				if (progressReporterHandle !== undefined) {
```

Comment:
I don't get it, this is a number but it looks like it isn't used anywhere else?

Reply:
Missed that while simplifying this change, fixed in https://github.com/microsoft/vscode/pull/212859. Thanks!